vendor/modules.txt

```diff
@@ -24,6 +24,7 @@ github.com/coreos/go-json
 # github.com/coreos/go-semver v0.3.0
 github.com/coreos/go-semver/semver
 # github.com/coreos/go-systemd v0.0.0-20191104093116-d3cd4ed1dbcf
+## explicit
```

Comment:
all this explicit comments came in automatically?
is that a new go feature?

Reply:
Yes, it came automatically. It a new go 1.14 feature, see - https://golang.org/doc/go1.14#go-command.